### PR TITLE
git-commit-graph: add page

### PR DESCRIPTION
**Pull request by bl-ue**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #3953

### CODE REVIEW

File: pages/common/git-commit-graph.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Write a commit-graph file for the packed commits in the repository's local `.git` directory:
+
+`git commit-graph write`
+
+- Write a commit-graph file containing all reachable commits:
+
+`git show-ref --hash | git commit-graph write --stdin-commits`
+
+- Write a commit-graph file containing all commits in the current commit-graph file along with those reachable from `HEAD`:
```
Comment (sbrl) [inline pages/common/git-commit-graph.md:14]: This sounds a bit confusing to me. Can you explain what this does please?

Comment (bl-ue) [reply pages/common/git-commit-graph.md:14]: Sure!  That came straight from the [git docs](https://git-scm.com/docs/git-commit-graph).

Comment (sbrl) [reply pages/common/git-commit-graph.md:14]: Right, but which part? Reading all of the docs for this might take a long time...?

Comment (bl-ue) [reply pages/common/git-commit-graph.md:14]: >Right, but which part? 

https://git-scm.com/docs/git-commit-graph

>Besides, if this came fromt he docs, it doesn't sound like the docs are particularly clear on the subject.

You're definitely right about that.  I think they focus more on Git itself :)

Comment (bl-ue) [reply pages/common/git-commit-graph.md:14]: Now, actually, I don't really even know what they intended there, so I don't know if I can improve it! ;P

I'll try though.

Comment (sbrl) [reply pages/common/git-commit-graph.md:14]: Oh, I see. `giit show-ref -s` seems to list a number of different commit hashes, which are then processed by the `commit-graph` subcommand.

**Review by sbrl — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**